blackbox/handlers/storage/s3.py

```diff
@@ -84,7 +84,8 @@ def sync(self, file_path: Path) -> None:
             self.client.upload_fileobj(
                 file_,
                 self.bucket,
-                file_path.name,
+                f"{file_path.name}.gz",
+                ExtraArgs={"ContentEncoding": "gzip"}
```

Comment:
What's the specific reason you decided to include this?

I did consider it back when, but when I tested it it didn't seem to do anything. Still, I don't think it hurts, either.

Reply:
I included it because I thought it would be better practice, I don't think S3 uses it for anything either; it justs returns it in the response metadata when you fetch the object.